spec/board_spec.rb

```diff
@@ -33,4 +33,52 @@
     expect {@board.update_grid(17, "X")}.to raise_error(Board::VoidMoveError)
   end
 
+  it "should place each row's content in an array" do
+    @board = Board.new(3)
```

Comment:
Same applies here re: DRY'er tests. You're calling `@board = Board.new(3)` many times.

Reply:
Done! the following tests are initialized to a difference Board size (2 or 4):
"should have a customizable size" @board = Board.new(4)
"should raise error if board size is less than 3" @board = Board.new(2)
"should raise an error if player's move is outside grid size range" @board = Board.new(4)
"should get all possible move combinations for a 4x4 board" @board = Board.new(4)

Reply:
`@board` also doesn't need to be an instance variable in here. Instance variables are useful when they need to be accessed outside of local scope, but in this case, `@board` is being reassigned in every test so we can change this to just a local variable `board`.

Reply:
@jayden  when I change board to be a local variable and I use a before hook to instantiate Board, my tests fail with this error: "undefined local variable or method `board'". But when I change board to be an instance variable all the tests pass again.

Reply:
Try using `let` blocks in your tests!

https://stackoverflow.com/questions/5974360/rspec-what-is-the-difference-between-let-and-a-before-block

Reply:
This needs to still be addressed (#1).

Reply:
Done!